jsons/Nations.json

```diff
@@ -2281,7 +2281,7 @@
     },
     {
         "name": "Poland",
-        "leaderName": "Caismir III",
+        "leaderName": "Casimir III",
```

Comment:
Note: this may or may not break other translations. I'll have to go through them and see if it was changed there as well. Give me some time (shouldn't take too much time)

Reply:
true, 
938c7f386b8f681f8a09b82f97bf8b557558aaea